docs/dev.rst

```diff
@@ -1,20 +1,522 @@
+
+..
+
+    Open the new ``dev`` org to begin development.
+
+.. code-block:: console
+
+    $ cci org browser dev
+
+
+Set the Capture State
+^^^^^^^^^^^^^^^^^^^^^
+
+To make changes to capture in an org, use the ``snapshot_changes`` task to create a snapshot. A snapshot tells the Salesforce CLI source tracking to set the org's current state as a baseline for changes to be made against.
+
+.. code-block:: console
+
+    $ cci task run snapshot_changes --org dev
+
+.. note:: There's no need to run ``snapshot_changes`` if you've set up a scratch org with any of these flows:
+
+    * ``dev_org``
+    * ``dev_org_namespaced``
+    * ``qa_org``
+    * ``regression_org``
+    * ``install_beta``
+    * ``install_prod``
+    
+    These flows all run ``snapshot_changes`` as their last step.
+
+The ``list_changes`` and ``retrieve_changes`` tasks detect new metadata but ignore any prior changes.
+
+To check that the snapshot was created successfully:
+
+.. code-block:: console
+
+    $ cci task run list_changes --org dev
+
+
+
+List Changes
+------------
+
+To see what components have changed in a target org:
+
+.. code-block:: console
+
+    $ cci task run list_changes --org dev
+
+.. note::
+    
+    This functionality relies on Salesforce's source tracking feature, so it's only available in scratch orgs.
+
+To include/exclude components from the list using the ``--include`` or ``--exclude`` option:
+
+.. code-block:: console
+
+    $ cci task run list_changes --org dev --include "test.*,another_regex" --exclude "something_to_exclude"
+
+The ``include`` and ``exclude`` patterns are matched against both the metadata type and name of the component.
+
+To include all changed components of specific types:
+
+.. code-block:: console
+
+    $ cci task run list_changes --org dev --types "CustomObject,CustomField"
+
+
+
+Retrieve Changes
+----------------
+
+The ``retrieve_changes`` task supports both ``sfdx`` and ``mdapi`` formatted source code. It utilizes the `SourceMember <https://developer.salesforce.com/docs/atlas.en-us.api_tooling.meta/api_tooling/tooling_api_objects_sourcemember.htm>`_ sObject to detect what has changed in an org, but also lets you be more selective regarding which components to retrieve when compared to the ``dx_pull`` task.
+
+Manual tracking of component versions also offers the possibility of retrieving some changes into one directory, and then running the task again to retrieve other changes into a different directory.
+ 
+To capture changes in an org:
+
+.. code-block:: console
+
+    $ cci task run retrieve_changes --org dev
+
+The task accepts ``include``, ``exclude``, and ``types`` options for filtering the list of changed components, and for scenarios where you don't want to retrieve all changed components.
+
+After the metadata is retrieved, the org snapshot updates so that the retrieved components are no longer included in ``list_changes``. 
+
+.. tip:: Avoid this by setting the ``snapshot`` option to False.
```

Comment:
Avoid this _______? This update? This exclusion? Would it be clearer to say: “To keep listing these retrieved components in ``list_changes``, set the ``snapshot`` option to False.

Reply:
"If you do not want a new snapshot to be created after running the `retrieve_changes` task, configure the `snapshot` option to `False` in in your project's `cumulusci.yml` file."